nixos/modules/services/backup/borgbackup.nix

```
@@ -191,10 +191,9 @@ in {
         options = {
 
           paths = mkOption {
-            type = with types; either path (listOf str);
+            type = with types; coercedTo path lib.singleton (listOf str);
```

```suggestion
            type = with types; coercedTo str lib.singleton (listOf str);
```
I seem to have forgotten this occurence when I last touched it.